grails 5 upgrade - geb issue

niravassar opened the pull request.

Commit:
grails 5 upgrade

sdelamo commented on complete/build.gradle:
```diff
@@ -6,8 +6,8 @@ buildscript {
     dependencies {
         classpath "org.grails:grails-gradle-plugin:$grailsVersion"
```
this should be decoupled from the grails version

sdelamo commented on complete/build.gradle:
```diff
@@ -6,8 +6,8 @@ buildscript {
     dependencies {
         classpath "org.grails:grails-gradle-plugin:$grailsVersion"
         classpath "gradle.plugin.com.github.erdi.webdriver-binaries:webdriver-binaries-gradle-plugin:2.1"
```
this should be 2.4

sdelamo commented on complete/build.gradle:
```diff
@@ -90,7 +90,10 @@ bootRun {
 
 webdriverBinaries {
-    chromedriver '95.0.4638.17'
+    chromedriver {
```
this should be the same version as grails_i18n guide

sdelamo commented on complete/build.gradle:
```diff
@@ -90,7 +90,10 @@ bootRun {
+        version = '95.0.4638.17'
+        architecture = 'X86'
+    }
     geckodriver '0.26.0'
```
this should be the same version as grails_i18n guide

sdelamo commented on complete/gradle.properties:
```diff
@@ -1,2 +1,6 @@
-grailsVersion=4.0.1
-gormVersion=7.0.2.RELEASE
+grailsVersion=5.0.0
```
5.0.1

sdelamo commented on complete/gradle.properties:
```diff
@@ -1,2 +1,6 @@
-grailsVersion=4.0.1
-gormVersion=7.0.2.RELEASE
+grailsVersion=5.0.0
+gormVersion=7.1.0.M5
```
7.1.0

puneetbehl committed:
Enable Workflow dispatch

puneetbehl committed:
Update githubactions-build.sh

puneetbehl committed:
Update gradle.properties

jamesfredley committed:
Replace dead publisher CI with sample-app gradle check workflow

Delete: githubactions-build.sh
Add: .github/workflows/gradle.yml (auto-detects Java from Gradle wrapper version)

(refs apache/grails-static-website#354)
